gateway/localStartExample.ps1

```diff
@@ -38,7 +38,7 @@ if ($BuildGateway) {
     Write-Host "Building gateway image: $GATEWAY_IMAGE_AND_TAG"
     docker build --tag "${GATEWAY_IMAGE_AND_TAG}" --no-cache .
 } else {
-    Set-Variable -Name GATEWAY_IMAGE_AND_TAG -Value "uwthalesians/gateway:0.2.0-build-129-branch-merge"
+    Set-Variable -Name GATEWAY_IMAGE_AND_TAG -Value "uwthalesians/gateway:0.3.0-build-186-branch-ocelot-login"
```

Comment:
how does the branch naming of images play into builds when we merge into develop?

Reply:
When there is a specific build number, like 186, that image is constant, meaning the same image will always be pulled. The reason for specifying a specific image tag here is to ensure the script will always be able to run using a known working build. 

The main way naming plays into develop is to allow a specific release build to be queued in azure pipelines which will then use the develop branch as the branch name in the tag. 

If you read the docker hub readme for [gateway](https://hub.docker.com/r/uwthalesians/gateway) or [mssql](https://hub.docker.com/r/uwthalesians/mssql) images the naming convention is explained.